iothub/device/src/ClientSettings/Http1TransportSettings.cs

```diff
@@ -47,5 +48,14 @@ public TransportType GetTransportType()
 
         /// <inheritdoc/>
         public IWebProxy Proxy { get; set; }
+
+        /// <remarks>
+        /// If not provided, an HTTP client will be created for you based on the other settings provided.
+        /// <para>
+        /// If provided, all other HTTP-specific settings (such as proxy, SSL protocols, and certificate revocation check)
+        /// on this class will be ignored and must be specified on this HttpClient instance.
+        /// </para>
+        /// </remarks>
+        public HttpClient HttpClient { get; set; }
```

Comment:
Seems to be missing a summary.

Can this HttpClient be used for SendEventAsync() as well?

Reply:
It can, and I'm adding an e2e test for that as well

Reply:
I have added the summary